src/com/pentalog/pentastagiu/homework2/HomeworkApp.java

```diff
@@ -1,7 +1,43 @@
 package com.pentalog.pentastagiu.homework2;
 
+import com.pentalog.pentastagiu.homework2.boardApp.Board;
+import com.pentalog.pentastagiu.homework2.boardApp.Users;
+
 public class HomeworkApp {
```

Comment:
sunt ok testele, ti-as sugera sa aplici schimbarile despre care ti-am scris si daca vrei sa iti mai exersezi skillurile poti incerca sa faci scenariile dinamice, in care sa citesti de la consola actiunea pe care vrea sa o faca userul si alti parametri de input

si ti-ar fi folositor sa faci putin debugging, sunt doar putine lucruri pe care trebuie sa le stii, dar cand ai probleme si nu iese codul cum te astepti o sa fie folositor sa poti sa vezi linie cu linie ce schimbari s-au facut in valorile variabilelor cu care lucrezi